test/browser_tests/page_objects/page_bureau-structure.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+function TheBureauStructurePage() {
+
+  this.get = function( ) {
+    browser.get( '/the-bureau/bureau-structure/' );
+  };
+
+  this.pageTitle = function() { return browser.getTitle(); };
+
+  this.sideNav = element( by.css( '.nav-secondary' ) );
```

Comment:
Actually this should probably just be named whatever design calls it, regardless of the class name. I can't remember what @duelj calls it, side nav or secondary nav?